R/list_distributions.R

```diff
@@ -24,6 +26,14 @@
 
 list_distributions <- function(delay_dist = "incubation", parameters = FALSE) {
 
+  delay_dist <- match.arg(
+    arg = delay_dist, 
+    choices = c(
+      "incubation", "onset_to_admission", "onset_to_death", "serial_interval"
+    ), 
```

Comment:
Could you put this directly in the function definition as you did in `extract_param()` please?

Reply:
Done bc4b5db42937115d74bc928b5082736a7a29f3a1